docs/UserGuide.md

```diff
@@ -64,6 +64,17 @@ Pupilist is a **desktop app for managing students details for private tutors**.
 
 </div>
 
+### Next session : `no command needed`
```

Comment:
hi u merged alr but i dont think you should put "no command needed" cos its abit confusing. rather, i think you should follow the format of the saving & editing data sections. you should prob shift it to the end as well since its essentially just a display feature, and i think the ones at the top should be left for things the user actually has to identify quickly.

Reply:
okkie will take a look after lunch
thank you @phyliciachristel